content/2_Bronze/Intro_Sets.mdx

```diff
@@ -755,3 +757,243 @@ Some of these problems can be solved by sorting alone, though sets or maps could
 make their implementation easier.
 
 <Problems problems="standard" />
+
+## Check Your Understanding
+
+<LanguageSection>
+<CPPSection>
+	
+<Quiz>
+  <Quiz.Question>
+    What is the time complexity of insertions, deletions, and searches in an ordered set?
+    <Quiz.Answer>
+      $\mathcal{O}(1)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N \log N)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $\mathcal{O}(\log N)$
+      <Quiz.Explanation>
+		Correct.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	#include <bits/stdc++.h>
```

Comment:
I don't think there's a need to include the header or main function

Reply:
K